pkg/service/test/test.go

```diff
@@ -218,6 +215,8 @@ func (t *tester) Test(path string, testReportPath string, appCmd string, options
 		return false
 	}
 	for _, sessionIndex := range initialisedValues.Sessions {
+		initialisedValues.Path = filepath.Join(path, sessionIndex)
+		initialisedValues.TestPath = filepath.Join(initialisedValues.Path, "tests")
```

Comment:
initialise path only once in the yam store and never specify it. and constants like tests in the respective functions not outside fucntions.